.github/workflows/main.yml

```diff
@@ -66,7 +66,7 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: clippy
-          args: --workspace --all-targets --all-features -- -D warnings -A clippy::upper-case-acronyms -A clippy::large-enum-variant
+          args: --workspace --all-targets --all-features -- -D warnings -A clippy::upper-case-acronyms -A clippy::large-enum-variant -W clippy::redundant_closure_for_method_calls
             
```

Comment:
I didn't want to switch on `pedantic` altogether but that one seems very useful because I can not think of a place where I  would rather like to have a redundant closure